torch/quantization/quantize.py

```diff
@@ -176,20 +177,33 @@ def add_quant_dequant(module):
         module._modules[name] = add_quant_dequant(child)
     return module
 
-def prepare(model):
-    r"""Prepares the model for calibration or training.
+def prepare(model, qconfig_dict=None, inplace=False):
+    r"""Prepares a copy of the model for quantization calibration or quantization-aware training.
 
-    The model will be attached with observer and quant dequant or fake quant
-    modules, and qconfig will be propagated.
+    Quantization configuration can be passed as an `qconfig_dict` or assigned preemptively
+    to individual submodules in `.qconfig` attribute.
 
-    Note that the model will be modified inplace but in case the input model
-    is a leaf model, a wrapped model will be returned.
+    The model will be attached with observer or fake quant modules, and qconfig
+    will be propagated.
 
     Args:
-        mod: input model
+        model: input model to be modified in-place
+        qconfig_dict: dictionary that maps from name or type of submodule to quantization
+            configuration, qconfig applies to all submodules of a given
+            module unless qconfig for the submodules are specified (when the
+            submodule already has qconfig attribute)
+        inplace: carry out model transformations in-place, the original model is destroyed
```

Comment:
Nit: is "destroyed" the right word here? It might look to users that if I do `prepare(model)` then the `model` is destroyed and I cannot use it anymore.